build.gradle

```diff
@@ -136,76 +132,49 @@ sourceSets {
     }
 }
 
-// All paths are relative to 'alpaca-java/sample_json/'
-// The 'path' can either be a file or directory, but keep in mind that
-// if it is a directory, all the subdirectories will have POJOs generated
-// sourceType is either 'json' or 'schema'
-// The generated POJOs will be in a package structure analogous to the path
+// The generated POJOs will be in a package structure analogous to the path in the 'sample_json/' directory
 // See: https://github.com/joelittlejohn/jsonschema2pojo/wiki/Reference
-def sampleJSONPOJOs = [
-        // alpaca/
-        ['path': 'alpaca/account/', 'sourceType': 'json'],
-        ['path': 'alpaca/asset/', 'sourceType': 'json'],
-        ['path': 'alpaca/bar/', 'sourceType': 'json'],
-        ['path': 'alpaca/calendar/', 'sourceType': 'json'],
-        ['path': 'alpaca/clock/', 'sourceType': 'json'],
-        ['path': 'alpaca/order/', 'sourceType': 'json'],
-        ['path': 'alpaca/position/', 'sourceType': 'json'],
-
-        // polygon/
-        ['path': 'polygon/aggregate/', 'sourceType': 'json'],
-        ['path': 'polygon/historic/quotes/', 'sourceType': 'json'],
-        ['path': 'polygon/historic/trades/', 'sourceType': 'json'],
-        ['path': 'polygon/last/', 'sourceType': 'json'],
-        ['path': 'polygon/meta/', 'sourceType': 'json'],
-        ['path': 'polygon/open_close/', 'sourceType': 'json'],
-        ['path': 'polygon/reference/', 'sourceType': 'json'],
-        ['path': 'polygon/snapshot/', 'sourceType': 'json'],
-        ['path': 'polygon/websocket/', 'sourceType': 'json']
-]
-
 task generatePOJOs() {
     final def targetDirectory = "${project.buildDir}/generated-sources/samplejson"
-    final def jsonSourceDirectory = 'sample_json/'
-    def first = true
-
-    sampleJSONPOJOs.each { propertyMap ->
-        def path = propertyMap.get('path')
-        def sourceType = propertyMap.get('sourceType')
-        def targetPackage = 'io.github.mainstringargs.domain.' +
-                path.substring(0, path.lastIndexOf('/')).replace('/', '.')
-
-        def jsonToPOJOTask = tasks.create('jsonToPOJOTask-' + targetPackage, GradleBuild) {
-            startParameter.projectProperties = ['source'         : jsonSourceDirectory + path,
-                                                'sourceType'     : sourceType,
-                                                'targetPackage'  : targetPackage,
-                                                'targetDirectory': targetDirectory]
-            buildFile = 'sampleJson2Pojo.gradle'
-            tasks = ['generateJsonSchema2Pojo']
-        }
+    final def jsonSourceDirectory = "${project.projectDir}/sample_json/"
```

Comment:
Love that you went to schemas.  What do you think about changing this directory name now that it is schema based?

Reply:
Schemas are great right!!! What were you thinking I should change the directory name to? Maybe: `schema_json`?

Reply:
Yes! schema_json would work